### ISSUE TITLE
Explicit unicode column names are converted to lowercase when using postgresql

### ISSUE THREAD

**Issue by majkinetor** [evidence-dev/evidence#478]
**Bug Description**

Serbian cyrilic:  `Опис` is converted to `опис`. 

```obrasci
select description as "Опис"
from forms f
```

![image](https://user-images.githubusercontent.com/85767/204261592-30fe1483-ad7b-4989-ac8a-3cf5b2d46319.png)

Serbian latin:

```obrasci
select description as "Šuma"
from forms f
```

![image](https://user-images.githubusercontent.com/85767/204261281-013a884f-48c6-46c9-adc6-31c560051010.png)

**Severity**
- [x] Low - I'm able to keep using Evidence as normal, but flagging this for the team to fix

**Expected Behavior**

**Workarounds**
There doesn't seem to be workaround

**Environment Information**
Operating System: Windows 10
Node version (`node -v`): 19.1.0
npm version (`npm -v`): 8.9.13
Package versions (`npm list --depth=0`): `@evidence-dev/evidence@5.0.12

**Database:**
- [x] Postgres

**Comment by archiewood** [comment]
Hi @majkinetor,
Confirm this does not happen with other connectors (eg SQLite).

I **think** the specific piece of code here is [this line](https://github.com/evidence-dev/evidence/blob/c0af5c4779293d1d43cd09bb840c6505809cc360/packages/postgres/index.cjs#L76)

@hughess Is there a specific reason we lowercase the keys like this?

**Comment by hughess** [comment]
@archiewood I actually used the same function for the postgres connector as we already had for the snowflake connector - I believe there was a reason to do that for snowflake but I'm not sure if postgres has the same requirement. 

@mcrascal was there a reason for lowercasing the snowflake column names in the `standardizeResult` function? 

If there's no specific reason for postgres, we should be good to remove that line

**Comment by hughess** [comment]
2 other ways we can solve this issue:
1. Change the function that formats column titles to correctly handle cyrillic input (I think this might be difficult to accomplish)
2. Allow an override for column titles in viz components - this exists in charts now (can override `xAxisTitle` and `yAxisTitle`), but doesn't exist yet for tables. It's something we're including in the new tables we're building, so that should be an option to use soon.
    - E.g., you could do something like `<Column id=опис title="Опис"/>`

**Comment by mcrascal** [comment]
As far as I recall, the [Snowflake API returns uppercased column names](https://community.snowflake.com/s/question/0D50Z00009SDHEoSAP/is-there-case-insensitivity-for-table-name-or-column-names), regardless of how you write your query. It was quite annoying to need to uppercase everything when referencing columns in components. Just a preference thing -- either we get all uppercase, or all lowercase.

**Comment by hughess** [comment]
Ok perfect, thanks.

I've just confirmed the behaviour of the postgres db package:
- Column names passed using double quotes are returned with the same cases as what's in the double quotes (e.g., `"One"` --> `One`)
- All other column names are returned in lowercase (e.g., `ONE` --> `one`)

So I think that means we're good to remove the `standardizeResult` function for the postgres connector!

**Comment by hughess** [comment]
It looks like even with the removal of the lowercasing, we'll have an issue with the title formatting function - it's picking up whatever the first A-Z character is and it uppercasing that. We'll need to get it to stop at the first character in the string:

<img width="608" alt="CleanShot 2022-12-12 at 12 38 35@2x" src="https://user-images.githubusercontent.com/12602440/207115329-d96c84d7-43fa-4f16-bd90-887f82294256.png">

**Comment by hughess** [comment]
As it turns out, there's an issue in our function for formatting titles. The regex in that function was filtering out the non A-Z characters. With that fixed, we get the following:
<img width="610" alt="CleanShot 2022-12-12 at 16 37 36@2x" src="https://user-images.githubusercontent.com/12602440/207160145-aad91748-b392-44d4-ba6a-a8ae269427ad.png">

**Comment by hughess** [comment]
@majkinetor I've added a fix for the title formatting issue in #522. 

Does #522 solve your problem, or only part of it? If needed, we can fix the lowercase issue in postgres, but I'm hesitant to change that at the moment in case of unintended consequences.

**Comment by majkinetor** [comment]
Good enough. Thank you.
